src/rgw/rgw_admin.cc

```diff
@@ -1,1011 +1,55 @@
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
 // vim: ts=8 sw=2 smarttab
 
-#include <errno.h>
+#include <cerrno>
```

Comment:
why is this needed?

Reply:
cerrno is a C++ header while errno.h is a C one. No difference in behavior.

Reply:
In general you should not mix changes it makes reviews easier. This should be in a separate commit.